src/main/scala/li/cil/oc/common/block/Cable.scala

```diff
@@ -61,13 +61,7 @@ class Cable(protected implicit val tileTag: ClassTag[tileentity.Cable]) extends
   override def getBoundingBox(state: IBlockState, world: IBlockAccess, pos: BlockPos): AxisAlignedBB = Cable.bounds(world, pos)
 
   override def addCollisionBoxToList(state : IBlockState, world : World, pos : BlockPos, bigBox : AxisAlignedBB, boxes : util.List[AxisAlignedBB], entity : Entity) = {
```

Comment:
`bigBox` should probably be renamed to match the MCP name of `entityBox`, for clarity